app/controllers/calls_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CallsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  #skip_before_filter :authenticate_user!, :only => "reply"	
+  
+  def begin
+  	boot_twilio
+  	beginCall = BeginCall()
+  	response = VoiceResponse(beginCall)
+    response.xml()
```

Comment:
Prefer omitting parenthesis when you have no arguments.

You should be using `render` here. `render xml: response.xml`

Reply:
What should be the behavior of `response.xml` ? 
Since before we had `render xml: response.to_s`
would the method `response.xml`  even be necessary?